portfolio/src/main/webapp/script.js

```diff
@@ -126,3 +126,37 @@ const addComments = async () => {
     );
   }
 };
+
+/*************************
+ *     RANDOM FACT 
+ ************************/
+async function loadFromCSV(mapObject){
+  d3.csv("/assets/markers.csv").then(function(data) {
```

Comment:
It looks like this is loading directly from CSV instead of the servlet -- is that intentional?

Reply:
yes this is intentional. should I change it to send a JSON instead?

Reply:
will follow up in Chat!